test-remote-cluster.sh

```diff
@@ -0,0 +1,9 @@
+# before, execute: export NO_HEADLESS=true && export CHROME_VERSION=$(/usr/bin/google-chrome-stable --version)
+
+export BRIDGE_KUBEADMIN_PASSWORD=4DGom-8hwk7-ZiVkZ-YzigC
+export BRIDGE_BASE_ADDRESS="https://console-openshift-console.apps.satwsin1-44.149.81.133.59.nip.io"
```

Comment:
Did you mean this check this in? We shouldn't have a random URL at least.

Reply:
yikes!  no, that is a dev script I have

Reply:
removed from PR